addons/website/views/snippets/s_company_team_detail.xml

```diff
@@ -0,0 +1,153 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_company_team_detail" name="Team Detail">
+    <section class="s_company_team_detail o_cc o_cc1 pt64 pb64">
+        <div class="container">
+            <h3>Meet our team</h3>
+            <p class="lead">Dedicated professionals driving our success</p>
+            <div class="row s_nb_column_fixed">
+                <div class="col-lg-4 pt32 pb32" data-name="Team Member">
+                    <div class="o_not_editable" contenteditable="false">
+                        <img class="o_editable_media img img-fluid rounded-circle" src="/web/image/website.s_company_team_image_1" alt="" style="width: 25% !important;"/>
+                    </div>
+                    <p>
+                        <br/>
+                    </p>
+                    <h4 class="h5-fs">James Mitchell</h4>
+                    <p class="text-muted">Chief Technical Officer (CTO)</p>
```

Comment:
Also, nitpicking but I think it's weird to have only one role with something inside parenthesis `(CTO)`